pkg/agent/metrics/prometheus.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"k8s.io/klog"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent"
+	"github.com/vmware-tanzu/antrea/pkg/agent/interfacestore"
+	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
+)
+
+type OVSStatManager struct {
+	ofClient     openflow.Client
+	OVSBridge    string
+	OVSTableDesc *prometheus.Desc
+}
+
+func (c *OVSStatManager) GetOVSStatistics() (
+	ovsFlowsByTable map[string]float64,
```

Comment:
For a single return value, maybe no need to break a newline and have extra parenthesis

Reply:
Done